main/acle.rst

```diff
@@ -404,6 +408,131 @@ Scalable Vector Extensions (SVE)
 ACLE support for SVE is defined in the Arm C Language Extensions for SVE
 document [SVE-ACLE]_ available on the Arm Developer Website.
 
+.. _ssec-NEONSVEBridge:
+
+NEON-SVE Bridge
+---------------
+
+The NEON-SVE Bridge adds intrinsics that enables setting NEON vectors
+into SVE vectors, and getting NEON vectors from SVE vectors. As the first 128
+bit subvector of an SVE vector is physically a NEON register,
+these intrinsics result in a no-op.
+
+These intrinsics are defined in the header file ``arm_extensions_bridge.h``
+and must be included after ``arm_neon.h`` and ``arm_sve.h``. These
+intrinsics are available in the ``aarch64`` state.
+
+Including this header file will define ``__ARM_SVE_NEON_BRIDGE`` as 1.
+
+``svset_neonq``
+~~~~~~~~~~~~~~~
+
+These intrinsics set NEON vector ``subvec`` into the first 128 bit subvector of
+the SVE vector ``vec``.
```

Comment:
Is it worth including a version of these to set the low 128-bits of an SVE vector and ignore the top bits entirely. I imagine we can do that with this existing proposal with e.g. `svset_neonq_s8(svundef_s8(), a)` but I don't know whether there is a preference for relying on compilers optimising that away into a no-op versus providing a dedicated set of intrinsics to handle this?

Reply:
`svset_neonq_s8(svundef_s8(), a)` should be able to handle this perfectly fine we think.